examples/trace/build.gradle

```diff
@@ -24,12 +23,11 @@ task fatJar(type: Jar) {
     with jar
 }
 
-
 description = 'Examples for Cloud Trace Exporter'
 
 dependencies {
     compile(libraries.opentelemetry_api)
     compile(libraries.opentelemetry_sdk)
     compile(libraries.google_cloud_trace)
-    compile project(':exporter-trace')
```

Comment:
👍

Reply:
One thing I'm not too sure about here is how will we deal with breaking changes. Will we have to update example only after exporter is released? That's probably fine, just untrivial to test before making a release.

Reply:
Yeah, the changes to the example project code for breaking changes will be on the `bump-version` branch, which is also where the version update is for the example project dependency, so it shouldn't block the release process. I think it should be okay to carry on with the release, and come back to the `bump-version` branch once the release is complete to fix up the example project code. WDYT?
I could add a comment in the `bump-version` section of the release instructions to say the above.

Reply:
Sounds good to me!